beetsplug/convert.py

```diff
@@ -100,8 +100,18 @@ def should_transcode(item, fmt):
             query, _ = parse_query_string(query_string, Item)
             if query.match(item):
                 return False
-    if config['convert']['never_convert_lossy_files'] and \
-            not (item.format.lower() in LOSSLESS_FORMATS):
+
+    try:
+        convert_lossy = not config['convert']['never_convert_lossy_files']
```

Comment:
This might be a little loose. Configs are handled through the `confuse` library? I couldn't find how it maps ex. "true" / "yes" / True (no quotes) in a config to boolean True / False.

Reply:
I think you can use `'never_convert_lossy_files' in config['convert']` to check whether the option has a setting.